examples/by_runtime_api_module/device_management.cpp

```diff
@@ -62,7 +62,7 @@ int main(int argc, char **argv)
 	// Attributes and properties
 	// ---------------------------
 
-	auto max_registers_per_block = device.get_attribute(cudaDevAttrMaxRegistersPerBlock);
+	auto max_registers_per_block = device.get_attribute(static_cast<cuda::device::attribute_t>(cudaDevAttrMaxRegistersPerBlock));
```

Comment:
Ah, it looks like the return type of get_attribute needs to change, there shouldn't be a cast here.

Reply:
I suspect as much, but since I currently don't have the opportunity to test this on a non-Windows machine I don't know if changing the return type would be ok.